interface/src/app/analyze/analyze.tpl.html

```diff
@@ -1,44 +1,52 @@
-<div>
-  <div id="search" ng-class="detail.showing ? 'with-detail':''">
-    <h1 class="page-header">
-      Analyze
-    </h1>
-    <p>
-      In this area, users will be able to search for an experiment or set of
-      samples to analyze, define analysis groups and generate a DA heatmap to
-      review ADAGE signature intensities for each sample.
-    </p>
-    <div id="search-results" class="col-sm-11">
-      <search placeholder="query text" results="analyze.query_results"></search>
-      <ul class="list-group">
-        <li ng-repeat="search_item in analyze.query_results"
-        ng-class="analyze.item_style(search_item)" class="list-group-item">
-          <a id="{{search_item.pk}}" ng-click="detail.show(search_item)">
-            <div class="media-left">
-              <span ng-if="search_item.item_type === 'experiment'"
-              class="badge">
-                {{ search_item.related_items.length }}
-              </span>
-            </div>
-            <div class="media-body">
-              <button type="button" class="btn"
-              ng-class="sb.hasItem(search_item)?'disabled':''"
-              ng-click="sb.addItem(search_item)">
-                <i class="fa fa-plus" aria-hidden="true"></i>
-              </button>
-              <h4 class="list-group-item-heading">
-                {{search_item.description}}
-              </h4>
-              <p class="list-group-item-text">{{ search_item.pk }}:
-                ...<span ng-bind-html="search_item.snippet"></span>...
-              </p>
-            </div>
-          </a>
-        </li>
-      </ul>
+<div class="page-header">
+  <h1>Analyze</h1>
+  <ml-model-view></ml-model-view>
+  <ml-model-validator model-id="modelInUrl" is-valid-model="isValidModel">
+  </ml-model-validator>
+</div>
+
+<div ng-if="isValidModel">
```

Comment:
I figured out another solution: on line 8 (my PR) of `analyze.tpl.html`, use `ng-show` instead of `ng-if`, it works too. You can test it.

Rene had a similar issue sometime ago. I think this is one of the key differences between `ng-show` and `ng-if`.